conf/application.conf

```diff
@@ -88,10 +88,17 @@ microservice {
         cdsSubscribeUrl =  "/customs/subscribe-for-cds"
     }
 
-      auth {
-          host = localhost
-          port = 8500
-        }
+    cds-reimbursement-claim {
+      protocol = http
+      host = localhost
+      port = 7501
+      claims-url = "/claim"
```

Comment:
need to add the domain for the BE service: `claims-url = "/cds-reimbursement-claim/claim"`